openedx_webhooks/jira/tasks/utils.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
Maybe, I don't know yet. We don't have any JIRA handlers yet. If it turns out there are useful utilities to be used by both tasks and handlers, then we can move them up to `jira.utils`.